### PR TITLE
Feature/animation

### PR DESCRIPTION
**Pull request by Albinpirela**
# Add Animations in the portfolio 📂 🗃️ 
Captura de pantalla (161)
Captura de pantalla (160)

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -459,9 +529,33 @@ button > span {
...
+  animation-fill-mode: forwards;
+}
+
+.blog a > button:not(:active) {
+  animation-name: animacion-inversa;
+  animation-duration: 3s;
+  animation-fill-mode: forwards;
+}
+
+@keyframes animacion-inversa {
```
Comment (GerardoGomezG) [inline styles.css]: Hello dude!

I've checked the animations of your portfolio. The one I liked most is with the hover action on any of the skills containers for desktop version.

Please, keep in mind that English language is preferred when coding, so I suggest you to translate "animacion-inversa".

**Review by GerardoGomezG — APPROVED** [review]
I really liked your animations effect @Albinpirela !

There's nothing wrong with functionality, just remember to write everything in English.

By the way, I also checked the links of your social networks. So i have some suggestions:
* Consider to delete the link to gmail.com related to your e-mail
* Your Twitter link isn't working. Consider to delete the "@home" part and just leave it like this: "https://twitter.com/Albinpirela1"
* There's an option to create a figma community profile and once created is where we address the link. Anyway, I suggest you to replace the figma info with your linkedin account link

I almost forget...
Consider adding a "Get my resume" button to your portfolio.

As no animation functionality is affected, I approve this PR.

File: styles.css (hunk 1 above)
Comment (Albinpirela) [reply styles.css]: ##@GerardoGomezG thanks for the suggestion and the changes are ready I hope you like them
Captura de pantalla (165)
Captura de pantalla (164)

**Review by GerardoGomezG — APPROVED** [review]
Well done pal!

You're ready for next stage developing...

Approved!

**Pull request merged**